_includes/docs/0.4.x/gettingstarted.html

```diff
@@ -39,7 +39,7 @@
     There are several ways you can get the MEAN.JS boilerplate:
   </p>
   <h3>Yo Generator</h3>
-  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use the development version.
+  <p>Although the main generator is released, the sub generators are still under development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use main generator.
```

Comment:
Do we want to refer to it as the "main generator" or is there another name we can use to make more sense? just maybe "meanjs generator" or something.

at the end of the line it reads "how to use main generator" probably should have a "the" there.